packages/ddp-client/common/urlHelpers.js

```diff
@@ -12,6 +12,10 @@ function translateUrl(url, newSchemeBase, subPath) {
     newSchemeBase = 'http';
   }
 
+  if (subPath !== "sockjs" && url.startsWith("/")) {
+    url = Meteor.absoluteUrl(url.substr(1));
+  }
```

Comment:
Thanks to @stubailo for [figuring this out](https://github.com/meteor/meteor/pull/9316/files#r149388945).

Reply:
I think a total rewrite of this function would be a great idea sometime soon, was going to be my next step for hack week. It's one of those functions I hate that pretends to be generic but actually only works for a very slim combination of inputs.